apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/InputReceiver/ReadQRCodeScreen.tsx

```diff
@@ -36,7 +38,7 @@ export const ReadQRCodeScreen = () => {
     return Promise.resolve(false)
   }
 
-  return <QRCodeScanner onRead={handleOnRead} />
+  return <QRCodeScanner onRead={handleOnRead} maskText={strings.addressReaderQrText} />
```

Comment:
if the client pass the `maskText` doesn't it mean that the mask should be enabled?

Reply:
I left the possibility in case it is necessary to have a mask without text. Let me know if it is necessary to stick with just that variable. Thanks!